README.md

```diff
@@ -24,6 +24,7 @@ Otherwise, the tool will download all `ocrd-tesserocr-recognize` models as well
 
 - clone this repository
 - (optional) [customize](#custom-workflows-and-data) QuiVer Benchmarks according to your needs
+- (optional) run `bash prepare.sh` to download sample data to the `gt` directory
```

Comment:
I'm not sure whether this optional in the current state. If I don't do this I get an error during `up` :  

`cat: '/app/gt/*//mets.xml': No such file or directory
quiver-benchmarks-benchmarks-1  | cp: cannot stat '/app/gt/*': No such file or directory
`

Is there a way to be able to start the container first and then run prepare?

Reply:
Yes, you could mount the script into the container and then run it. And yes, `(optional)` is a bit misleading. I thought it's "either customizing or preparation script".